schema.sql

```diff
@@ -9,4 +9,26 @@ CREATE TABLE animal (
     weight_kg decimal
 );
 
-Alter TABLE animal ADD species char(20);
\ No newline at end of file
+Alter TABLE animal ADD species char(20);
```

Comment:
- Please, kindly ensure your animal table is pluralized as `animals` because the table has many animals. please kindly update this.